docs/eigenda/operator-guides/run-a-node/run-with-docker.mdx

```diff
@@ -0,0 +1,195 @@
+---
+title: Run with Docker
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Run a node using docker
+
+We provide an Operator Setup Repository which contains various templates that make it easy to run an EigenDA node using docker and docker compose. Operators wishing to make use of other setups can use the docker-compose-based setup as a template for constructing their own custom setups. 
+
+To proceed with these instructions, be sure to ensure that you have installed docker on your system.
+- [Docker Engine on Linux](https://docs.docker.com/engine/install/ubuntu/).
+
+
+## EigenDA Node Configuration
+
+#### Clone the Oprator Setup Repo and populate the environment variables
+
+Run the following commands to clone the Oprator Setup Repo and create a new environment file from the provided template. 
+
+<Tabs groupId="network">
+  <TabItem value="mainnet" label="Mainnet">
+    ```
+    git clone https://github.com/Layr-Labs/eigenda-operator-setup.git
+    cd eigenda-operator-setup/mainnet
+    cp .env.example .env
+    ```
+  </TabItem>
+  <TabItem value="holesky" label="Holesky">
+    ```
+    git clone https://github.com/Layr-Labs/eigenda-operator-setup.git
+    cd eigenda-operator-setup/holesky
+    cp .env.example .env
+    ```
+  </TabItem>
+</Tabs>
+
+The provided `.env` contains many default configuration settings for the node. All sections marked with `TODO` must be updated to match your environment. We recommend that operators follow the steps in the next section to configure their node to run without access to their ECDSA private key. 
+
+:::info
+As described [here](../registration/), the ECDSA and BLS keys needed to populate your `.env` can be obtained via the process of registering as an operator for EigenLayer. 
+:::
+
+:::warning
+Do not modify the docker-compose.yml file. If you choose to modify this file, unexpected behaviors can result.
+:::
+
+#### Download the Structured Reference String (SRS)
+
+EigenDA makes use of a structured reference string (SRS) as part of its KZG verification. Run the following command to download all required files into the `eigenda-operator-setup/resources` directory. 
```

Comment:
I think there is another confusion . we are asking them to run `srs_setup.sh` here but in the above step we have in `holesky/mainnet` folder , and this script is in `eigenda-operator-setup` root folder - so I am pretty sure someone will get confused that there is no srs_setup. 

Maybe we should do something like
Step 1 clone repo
step 2 - do srs setup
step 3 - go to network folder and copy env

Reply:
Good point. I implemented your suggestion.